web/modules/custom/paragraph_align/paragraph_align.routing.yml

```diff
@@ -0,0 +1,7 @@
+paragraph_align:
+  path: '/paragraph/page'
```

Comment:
Will be better: /admin/paragraphs/align

1) Management pages should be in /admin section
2) Path should be simple and describe the goal

Reply:
Isn't solved